clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleRegionalServerGroupCachingAgent.groovy

```diff
@@ -532,6 +533,12 @@ class GoogleRegionalServerGroupCachingAgent extends AbstractGoogleCachingAgent i
           }
         }
       }
+
+      String pageToken = autoscalerAggregatedList.getNextPageToken()
+      if (pageToken) {
+        batchRequest.queue(compute.autoscalers().aggregatedList(project).setPageToken(pageToken),
+          new AutoscalerAggregatedListCallback(serverGroups: serverGroups, batchRequest: batchRequest))
```

Comment:
Is there a reason we can't just pass the same `AutoscalerAggregatedListCallback` instance here, instead of creating a new one?

Reply:
Nope good point.